cognito-post-authentication/src/main/java/no/unit/nva/cognito/PostAuthenticationHandler.java

```diff
@@ -1,157 +1,134 @@
+
 import com.amazonaws.services.cognitoidp.AWSCognitoIdentityProviderClient;
-import com.amazonaws.services.cognitoidp.model.AdminAddUserToGroupRequest;
-import com.amazonaws.services.cognitoidp.model.AdminUpdateUserAttributesRequest;
 import com.amazonaws.services.cognitoidp.model.AttributeType;
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import java.net.http.HttpClient;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.stream.Collectors;
+import no.unit.nva.cognito.model.Event;
+import no.unit.nva.cognito.model.Role;
+import no.unit.nva.cognito.model.User;
+import no.unit.nva.cognito.model.UserAttributes;
 import no.unit.nva.cognito.service.CustomerApi;
 import no.unit.nva.cognito.service.CustomerApiClient;
-import no.unit.nva.cognito.util.OrgNumberCleaner;
+import no.unit.nva.cognito.service.UserApiClient;
+import no.unit.nva.cognito.service.UserService;
 import nva.commons.utils.Environment;
 import nva.commons.utils.JacocoGenerated;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class PostAuthenticationHandler implements RequestHandler<Map<String,Object>, Map<String,Object>> {
-
-    public static final String USER_ATTRIBUTES = "userAttributes";
-    public static final String USER_NAME = "userName";
-    public static final String USER_POOL_ID = "userPoolId";
+public class PostAuthenticationHandler implements RequestHandler<Event, Event> {
 
-    public static final String CUSTOM_AFFILIATION = "custom:affiliation";
     public static final String CUSTOM_APPLICATION_ROLES = "custom:applicationRoles";
     public static final String CUSTOM_APPLICATION = "custom:application";
     public static final String CUSTOM_CUSTOMER_ID = "custom:customerId";
-    public static final String CUSTOM_FEIDE_ID = "custom:feideId";
     public static final String CUSTOM_IDENTIFIERS = "custom:identifiers";
-    public static final String CUSTOM_ORG_NUMBER = "custom:orgNumber";
 
+    public static final String COMMA_DELIMITER = ",";
     public static final String FEIDE_PREFIX = "feide:";
     public static final String NVA = "NVA";
-    public static final String PUBLISHER = "Publisher";
-    public static final String PUBLISHER_GROUP = "PublisherGroup";
-    public static final String STAFF = "staff";
-    public static final String REQUEST = "request";
-    public static final String COMMA_DELIMITER = ",";
 
-    private final CustomerApi customerApi;
-    private final AWSCognitoIdentityProvider awsCognitoIdentityProvider;
+    public static final String NOT_FOUND_ERROR_MESSAGE = "No customer found for orgNumber: ";
+
+    private UserService userService;
+    private CustomerApi customerApi;
 
     private static final Logger logger = LoggerFactory.getLogger(PostAuthenticationHandler.class);
 
+
     /**
```

Comment:
in checkstyle config find the module "MissingJavadocMethod" and remove the string "CTOR_DEF" to avoid the need for noisy comments in constructors.